graphrag/query/input/retrieval/relationships.py

```diff
@@ -19,6 +20,8 @@ def get_relationships_from_graphdb(query:str,selected_entity_names:list[str],gra
             "prop_selected_entity_names": selected_entity_names,
         }
     )
+    time.sleep(5)
```

Comment:
I don't think we should be adding sleeps here

Reply:
When testing I noticed that I got a too many requests error without this on local emulator